test/iht/views/application/assets/insurancePolicy/InsurancePoliciesOverviewViewTest.scala

```diff
@@ -16,50 +16,34 @@
 
 package iht.views.application.assets.insurancePolicy
 
-import iht.models.application.assets.InsurancePolicy
 import iht.testhelpers.CommonBuilder
-import iht.views.ViewTestHelper
+import iht.views.application.{ApplicationPageBehaviour, CancelComponent, Guidance}
 import iht.views.html.application.asset.insurancePolicy.insurance_policies_overview
-import play.api.i18n.MessagesApi
 import play.api.i18n.Messages.Implicits._
```

Comment:
We need more tests here, for the content of the overview items. See the  qualifying bodies overview view tests (in temp-master) for example.